.github/workflows/build-test.yml

```diff
@@ -88,6 +88,15 @@ jobs:
         crate: cargo-make
         version: latest
 
+    - name: Install Python
+      uses: actions/setup-python@v1
+    - name: Install Sphinx
+      uses: BSFishy/pip-action@v1
+      with:
+        packages: |
+          sphinx
+          sphinx-rtd-theme
```

Comment:
Suggestion: Wire this up with the artifacts CI so that the ReadTheDocs previews get uploaded alongside the Rustdoc previews.

Reply:
Mostly because messing with CI takes a lot of pushes and it's better to decouple that from other work